lambify/docker-compose.yml

```diff
@@ -4,10 +4,15 @@ services:
     build:
       context: '.'
       dockerfile: Dockerfile-build
+    entrypoint: ./bin/entrypoint
     environment:
       - RAILS_ENV=${RAILS_ENV-development}
       - SAM_CLI_TELEMETRY=0
       - AWS_PROFILE=${AWS_PROFILE-default}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION-us-east-1}
```

Comment:
Pass all the AWS environment variables for all docker compose run scripts and let the `entrypoint` clean it up as needed. This also has the upset of allowing developers to use cloud native resources for dev/test via `AWS_PROFILE` with Dotenv.